exporters/otlp/src/test/java/io/opentelemetry/exporters/otlp/MetricAdapterTest.java

```diff
@@ -52,7 +52,7 @@ void toProtoLabels() {
     assertThat(MetricAdapter.toProtoLabels(Labels.of("k", "v")))
         .containsExactly(StringKeyValue.newBuilder().setKey("k").setValue("v").build());
     assertThat(MetricAdapter.toProtoLabels(Labels.of("k1", "v1", "k2", "v2")))
-        .containsExactly(
+        .containsExactlyInAnyOrder(
```

Comment:
You're the metric expert :) But at a glance it seems like order is supposed to matter here?

Reply:
"expert". actually, I think this can be reverted, now that I reverse the list after de-duping. I'll look